### ISSUE TITLE
Can't install version 0.3

### ISSUE THREAD

**Issue by cesalazar** [terminal-labs/lektor-jinja-content#7]
When executing `lektor plugin add lektor-jinja-content` and then `lektor serve` the following error is displayed:

```
Collecting lektor-jinja-content==0.3
  Could not find a version that satisfies the requirement lektor-jinja-content==0.3 (from versions: 0.1, 0.2)
No matching distribution found for lektor-jinja-content==0.3
```

Seems like the release is not published: https://pypi.org/project/lektor-jinja-content/#files

**Comment by nixjdm** [comment]
This is curious to me. That link shows for me that v0.3 is published (in May too), and I have no problem downloading it and installing it. Did you change your source away from pip's default? Maybe pypi had a bug or outage? If that's the case, it seems fixed now.

I'm going to close this since I can't reproduce it. If you still have trouble, I'll repoen it, but I'll need some more info to go off of.

**Comment by nixjdm** [comment]
You may also want to try the just-released version 0.4.1.

**Comment by cesalazar** [comment]
Sorry for the late response, swamped at the moment. I'll give it a try, thank you!
